cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -130,25 +130,25 @@ CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
 FOR (n)
 ON EACH labels(n)
 ----
-| Create a lookup index on nodes.
+| Create a node label lookup index.
 
-Lookup indexes do not support any `OPTIONS` values.
+Token lookup indexes do not support any `OPTIONS` values.
 
 | [source, cypher, role=noplay]
 ----
 CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
 FOR ()-[r]-()
 ON [EACH] type(r)
 ----
-| Create a lookup index on relationships.
+| Create a relationship type lookup index.
 
-Lookup indexes do not support any `OPTIONS` values.
+Token lookup indexes do not support any `OPTIONS` values.
 
 | [source, cypher, role=noplay]
 ----
 DROP INDEX index_name [IF EXISTS]
 ----
-| Drop an index, either a node, a relationship, or a lookup index.
+| Drop an index, either a btree, fulltext, or token lookup index.
```

Comment:
I see a little inconsistency if we are using btree or b-tree here. In operations manual we use b-tree in text unless we are specifically referring to syntax or name in code such as `SHOW INDEXES#BTREE`, `org.neo4j.graphdb.schema.IndexType#BTREE`, `native-btree-1.0`
https://neo4j.com/docs/operations-manual/current/performance/index-configuration/

Reply:
that should probably be unified yes, I think we have a mix here

Reply:
I think that the old code hopefully is consistent so for now I just updated the new places :)